app/src/main/java/com/github/studydistractor/sdp/data/UserData.kt

```diff
@@ -11,11 +11,13 @@ package com.github.studydistractor.sdp.data
  * @constructor create a user
  */
 data class UserData(
-    var id: String? = null,
+    var id: userId? = null,
     var firstname: String? = null,
     var lastname: String? = null,
     var phone: String? = null,
     var birthday: Int? = null,
     var score: Int?= null
 )
 
+typealias userId = String
```

Comment:
Won't this cause some troubles with the DB ?

Reply:
No why would it cause troubles ? But anyway the point was resolved